tests/common/constants.py

```diff
@@ -1,3 +1,9 @@
 VLAN_SUB_INTERFACE_SEPARATOR = "."
 # default port mapping mode for storage backend testbeds
 PTF_PORT_MAPPING_MODE_DEFAULT = "use_sub_interface"
+TOPO_KEY = "topo"
+NAME_KEY = "name"
+# field in mg_facts to flag whether it's a backend topology or not
+IS_BACKEND_TOPOLOGY_KEY = "is_backend_topology"
+# a topology whos name contains 'backend' will be considered as a backend topology
+BACKEND_TOPOLOGY_FLAG = "backend"
```

Comment:
Rename this variable. 'FLAG' suffix is misleading. Generally the term 'flag' is to indicate a boolean value

Reply:
Done, currently use 'IND'(indicator) instead of 'FLAG'